service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/request1.request2.response1.response2.rpt

```diff
@@ -72,7 +72,7 @@ read [0x53 0x43 0x48 0x45 0x4D 0x45 0x20 0x3D 0x20 0x22 0x6A 0x61 0x76 0x61 0x73
 # Response 2
 read "HTTP/1.1 404 Not Found\r\n"
 read "Content-Type: text/html\r\n"
-read /Date: .+/ "\r\n"
+read /Date: .+"\r\n"/
```

Comment:
In this PR, you are using two ways to match Date header. can we use the first one ?
```
read /Date: .*\r\n/
read /Date: .+"\r\n"/
```